### PR TITLE
CiscoMeraki

### PR DESCRIPTION
**Pull request by randqmasters**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: CiscoMeraki/.pack-ignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[file:README.md]
```
Comment (evisochek) [inline CiscoMeraki/.pack-ignore:1]: Why do we need this?

File: CiscoMeraki/.secrets-ignore [hunk 2]
```diff
@@ -0,0 +1 @@
+11.11.1.111
```
Comment (evisochek) [inline CiscoMeraki/.secrets-ignore:1]: Why do we need this?

File: CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "Cisco_Meraki_raw": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      },
+  }
```
Comment (evisochek) [inline CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json:7]: Missing bracket, the JSON format is invalid

File: CiscoMeraki/.pack-ignore (hunk 1 above)
Comment (randqmasters) [reply CiscoMeraki/.pack-ignore:1]: In previous PRs, we received your comments to add this file.
/.pack-ignore
  See - Cisco SMA
The need for this is at your discretion.

File: CiscoMeraki/.secrets-ignore (hunk 2 above)
Comment (randqmasters) [reply CiscoMeraki/.secrets-ignore:1]: In previous PRs, we received your comments to add this file.
/.secret-ignore
  See - Cisco SMA
The need for this is at your discretion.

File: CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json (hunk 3 above)
Comment (randqmasters) [reply CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json:7]: We compared it to cisco SMA PR and did not see a  bracket, please correct me if I am wrong. 
{
    "Cisco_SMA_raw": {
      "_raw_log": {
        "type": "string",
        "is_array": false
      },
  }

File: CiscoMeraki/.secrets-ignore (hunk 2 above)
Comment (randqmasters) [reply CiscoMeraki/.secrets-ignore:1]: I deleted the irrelevant files.

File: Packs/CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+    "Cisco_Meraki_raw": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      },
```
Comment (evisochek) [inline Packs/CiscoMeraki/ModelingRules/CiscoMerakiModelingRules/CiscoMerakiModelingRules_schema.json:6]: the comma should be in the end

**Review by evisochek — APPROVED** [review]

**Pull request merged**